tests/e2e/transaction_queue/test_transaction_queue.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+
+import multiprocessing
+import sys
+import threading
+import time
+from typing import List
+
+import mgclient
+import pytest
+from common import connect, execute_and_fetch_all
+
+# Utility functions
+# -------------------------
+
+
+def get_non_show_transaction_id(results):
+    """Returns transaction id of the first transaction that is not SHOW TRANSACTIONS;"""
+    for res in results:
+        if res[2] != ["SHOW TRANSACTIONS"]:
```

Comment:
extract SHOW TRANSACTIONS into constant above

Reply:
Why :joy: ? Then it makes sense to change all queries to constants and I get a mess. + I think that SHOW TRANSACTIONS is not often used to make it a constant but pls. correct me if I am wrong.